scripts/DDL.sql

```diff
@@ -39,13 +39,24 @@ CREATE TABLE jobs.base_locations(
 CREATE TABLE jobs.users(
 	upn text PRIMARY KEY,
 	display_name text,
-	last_entrance timestamp,
+	last_entrance timestamp
+);
+
+CREATE TABLE jobs.user_resume(
+	upn text PRIMARY KEY,
 	rank_id int REFERENCES jobs.standards(id) ON DELETE CASCADE,
-	role_id int REFERENCES jobs.roles(id) ON DELETE CASCADE,
+	current_role_id int REFERENCES jobs.roles(id) ON DELETE CASCADE,
```

Comment:
what I suggest is a table called `previous_jobs` which contains all the previous roles of the user, instead of role - a user have qualification which will be added on my merge

Reply:
There is already a `previous_jobs` table